### DISCUSSION TITLE
Token Authentication Failed

### DISCUSSION THREAD

**Discussion by mirza019** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#445] [Q&A]
**Assistant failed to fetch the tokens from server. Either the auth code is invalid or the rate limit might have exceeded.
Try selecting a different Google Account.
Error: Error getting tokens**
![Capture](https://user-images.githubusercontent.com/45270816/124558780-c77abf80-de5c-11eb-8a8d-b4c202679efa.PNG)
![gg](https://user-images.githubusercontent.com/45270816/124558790-c8abec80-de5c-11eb-914d-a3aa9c452a9d.PNG)

After submitting the json file google ass. relaunched and took me to the auth page.. i got the token and copied it. when I used to paste it , then it shows the above mentioned error
Please solve this issue

**Comment by im-coder-lg** [comment]
Try using only one Google Account(the one you used for the project)

**Comment by im-coder-lg** [comment]
Check the FAQ [<kbd>here</kbd>](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#why-does-the-get-token-screen-fail-saving-tokens)

**Reply by garfolino** [reply]
followed the steps to the letter and still can't get past the 403 error

**Reply by im-coder-lg** [reply]
Dude, check properly. https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#why-am-i-getting-error-403-restricted_client is the link, and moreover, when you are on the OAuth screen, check the email(s) properly.
